backend/src/app.ts

```diff
@@ -34,6 +35,11 @@ export const router = Router<Router.HTTPVersion.V2>({ maxParamLength: 500 })
 router.get('/readinessProbe', readiness)
 router.get('/livenessProbe', liveness)
 router.get('/ping', respondOK)
+router.all('/api/v1/query', observabilityProxy)
+router.all('/api/v1/query_range', observabilityProxy)
+router.get('/api/v1/targets', observabilityProxy)
+router.get('/api/v1/rules', observabilityProxy)
+router.get('/api/v1/label', observabilityProxy)
```

Comment:
`/api` is used for k8s APIs... these don't seem to clash, but I wonder if it would be better to use a different route and rewrite in the proxy?

Reply:
Reading through this documentation, I *think* the router matches static routes first and places lowest priority on the wildcard routes, so it shouldn't have clashes.

Reply:
https://www.npmjs.com/package/find-my-way#match-order

Reply:
What I meant by "clashes" is that these routes take precedence over the wildcard, so this would be an issue if `query` or `rules`, for example, were k8s APIs that we want to access. We are lucky at the moment that there is no overlap, but in general I think it's bad practice to have 2 different backends serving stuff under `/api/v1`.

Reply:
Gotcha, I can update this to use an `/observability` route instead of the current `api/v1/...`